Sources/BraintreePayPal/BTPayPalAnalytics.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+class BTPayPalAnalytics {
```

Comment:
question: is there a benefit to this being a `class` instead of an `enum`?

Reply:
We opted for class until the SDK is in Swift - initially discussed here: https://github.com/braintree/braintree_ios/pull/959#discussion_r1123439103